### PR TITLE
Api/1200/storage system

### PR DESCRIPTION
**Pull request by sijeesh**
### Description
Teste storage system resource with the latest API versions 800, 1000, 1200

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -175,6 +175,18 @@
...
+|<sub>/rest/storage-systems/{id}</sub>                                                    | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/storage-systems/{id}/managedPorts</sub>                                       | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |
+|<sub>/rest/storage-systems/{id}/managedPorts/{portId}</sub>                              | GET      | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |
+|<sub>/rest/storage-systems/{id}/reachable-ports</sub>                                    | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
```
Comment (soodpr) [inline endpoints-support.md]: I don't see support for this endpoint in puppet.

File: endpoints-support.md [hunk 2]
```diff
@@ -175,6 +175,18 @@
...
+|<sub>/rest/storage-systems/{id}/templates</sub>                                          | GET      | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
```
Comment (soodpr) [inline endpoints-support.md]: I don't see support for this endpoint in puppet.

File: endpoints-support.md (hunk 1 above)
Comment (sijeesh) [reply endpoints-support.md]: Updated the file.

File: endpoints-support.md (hunk 2 above)
Comment (sijeesh) [reply endpoints-support.md]: Updated the file

**Review by AnushaY1916 — APPROVED** [review]

**Review by soodpr — APPROVED** [review]
LGTM 👍

**Pull request merged**